core/src/main/java/org/fao/geonet/kernel/setting/Settings.java

```diff
@@ -112,6 +112,7 @@
     public static final String SYSTEM_HARVESTING_MAIL_TEMPLATE_WARNING = "system/harvesting/mail/templateWarning";
     public static final String SYSTEM_HARVESTING_MAIL_TEMPLATE_ERROR = "system/harvesting/mail/templateError";
     public static final String SYSTEM_HARVESTING_MAIL_TEMPLATE = "system/harvesting/mail/template";
+    public static final String SYSTEM_HARVESTING_TIMEZONE = "system/harvesting/timeZone";
```

Comment:
Why not setting a global timezone? This could be a start of fixing https://github.com/geonetwork/core-geonetwork/issues/1970. We also have other cron tasks that are not harvester and may also be affected by that change?

Reply:
`QuartzSchedulerUtils.getTrigger(..., TimeZone tz)` is only used in `AbstractParams.getTrigger()` function, so no other jobs are affected apart of the harvesters.

Reply:
> Why not setting a global timezone? This could be a start of fixing #1970

It would be a bit confusing if we set as a global timezone and we don't fix all the issues described in #1970 yet, wouldn't it?

Reply:
It could be a good start (and if we don't have resources to do more than the harvesting) let's add in the settings help text that the timezone only applies to harvester schedule for now ?

Reply:
+1 to use a global timezone setting
i wonder if it makes sense if specific users would like to set the timezone specific to their local situation (for global portals)
having it as a global setting first would allow us to specify it for specific users later?